articles/libraries/lock-android/index.md

```diff
@@ -257,18 +257,15 @@ For a full list of Lock's configuration options, check out the [Lock for Android
 
 For descriptions of common error messages, check out the [Error Messages](/libraries/error-messages) page. Also, if your callback receives an `AuthenticationException` you can check [source](https://github.com/auth0/Auth0.Android/blob/master/auth0/src/main/java/com/auth0/android/authentication/AuthenticationException.java) to learn how to identify each error scenario.
 
-## Additional documents
-
-<ul>
-<% cache.find('articles/libraries/lock-android', {sort: 'toc_title'}).forEach(article => { %>
-  <% if (article.toc_title) { %>
-  <li>
-    <span><a href="<%- article.url %>"><%- article.toc_title %></a>
-    <% if (article.description) { %>
-      - <%- article.description %>
-    <% } %>
-    </span>
-  </li>
-  <% } %>
-<% }); %>
-</ul>
+## Next Steps
+
+::: next-steps
+
```

Comment:
The max, per guidelines, is 5 items. Can you remove one? Logout looks like a good candidate. Also the whitespace is note required inside the `next-steps` element